Rakefile

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env rake
+
+desc "Update vendor/cookbooks"
+task :vendor do
+  puts "Vendoring cookbooks"
+  `rm -rf vendor`
+  `berks update`
+  `berks vendor vendor/cookbooks`
+end
+
+desc "Package cookbooks into a chef-solo tarball"
+task :package => :vendor do
+  `mkdir -p vendor/cookbooks/conjur`
+  `cp -r metadata.rb Berksfile Berksfile.lock CHANGELOG.md chefignore README.md attributes recipes spec vendor/cookbooks/conjur`
+  version=`git describe --tags --abbrev=0`.strip
```

Comment:
Can you expand on this? This is what we've been doing to packaged
conjur-ssh.

On Mon, May 4, 2015 at 12:01 PM Rafał Rzepecki <notifications@github.com>
wrote:

> In Rakefile
> <https://github.com/conjur-cookbooks/conjur/pull/2#discussion_r29596694>:
>
> > @@ -0,0 +1,20 @@
> > +#!/usr/bin/env rake
> > +
> > +desc "Update vendor/cookbooks"
> > +task :vendor do
> > +  puts "Vendoring cookbooks"
> > +  `rm -rf vendor`
> > +  `berks update`
> > +  `berks vendor vendor/cookbooks`
> > +end
> > +
> > +desc "Package cookbooks into a chef-solo tarball"
> > +task :package => :vendor do
> > +  `mkdir -p vendor/cookbooks/conjur`
> > +  `cp -r metadata.rb Berksfile Berksfile.lock CHANGELOG.md chefignore README.md attributes recipes spec vendor/cookbooks/conjur`
> > +  version=`git describe --tags --abbrev=0`.strip
>
> I don't think we want --abbrev here. This will make the version release
> not always match the tag.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/conjur-cookbooks/conjur/pull/2/files#r29596694>.
>